lib/websocket_server.rb

```diff
@@ -2,6 +2,8 @@ class WebsocketServer
   Pairing = Struct.new(:is_ws, :proxy)
 
   def initialize
+    @logger = Rails.env.production? ? $websocket_log : Logger.new(STDOUT)
```

Comment:
Ah, so maybe you can pass in a logger to initialize and have the routes pass in STDOUT?  Otherwise, initialize could set logger to $websocket_log if it's defined or create a STDOUT logger.